Frends.Oracle.ExecuteQuery/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [1.1.0] - 2022-10-27
```

Comment:
Since task parameters have completely changed I would say that you should increase the major version instead of minor version.